test/views/home/vatReturnPeriodViewSpec.scala

```diff
@@ -79,15 +79,15 @@ class VatReturnPeriodViewSpec extends PlaySpec with GuiceOneAppPerSuite with Vat
     }
 
```

Comment:
I think we should get rid of this as well. Also from tests (uk.gov.hmrc.play.views.html.helpers.) I am doing it for other pages. Could remove any usage of it.